app/client/views/paper.js

```diff
@@ -196,10 +202,12 @@ CachedPaper.prototype._update = function () {
 module.exports = (result, state, emit) => {
   let el = elementcache[result.paper.key]
   if (el) {
+    el._query = state.search.querystring
```

Comment:
`state.search.querystring` is the raw query as typed - this can include partial tag search, which might break highlighting.

using `state.search.query` instead gets the pre-cleaned query without tag parts.